x/examples/website-wrapper-app/.scripts/build_mobileproxy.sh

```diff
@@ -15,29 +15,24 @@
 # limitations under the License.
 
 PLATFORM="$1"
-TAG="${2:-"x/v0.0.1"}"
-OUTPUT="${3:-output}"
+OUTPUT="${2:-output}"
```

Comment:
I'd do it this way, so one can set the OUTPUT anywhere they want. It also cleans up all the references of OUTPUT in your code:
```suggestion
OUTPUT="${2:-$(pwd)/output}"
```